packages/cli/test/scripts/verify.test.js

```diff
@@ -77,7 +77,11 @@ contract('verify script', function () {
       await assertVerify(contractAlias, { network, networkFile: this.networkFile, remote: 'invalid-remote' }, /Invalid remote/)
     })
 
-    it('throws error if contract could not be verified', async function () {
+    it('throws error if specifying anything but mainnet as a network for etherchain', async function() {
+      await assertVerify(contractAlias, { network, networkFile: this.networkFile, remote: 'etherchain' }, /Invalid network/)
+    })
+    
+    /*it('throws error if contract could not be verified on etherchain', async function () {
```

Comment:
We want these test cases to be working, and we should also add tests covering the new etherscan implementation. It would be a good idea to restructure the `contract verification` tests to something like:

```javascript
describe('validations', function() {
  // already existent tests 
})

describe('contract verification', function() {
  describe('against etherscan', function() {
    // new tests**
  })
  describre('against etherchain', function() {
   // here we should move the old tests we have for etherchain. 
  })
})
```
** We should at least cover the same cases we are covering for etherchain.

Last but not least: you can fix these commented tests by adding the `network: 'mainnet'` option in the object parameter :wink:.